backend/appwrite-functions/announcement/test.js

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: MIT
+// SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+const axios = require('axios')
+
+const auth_header = {
+    'X-Appwrite-Project': process.env.APPWRITE_PROJECT,
+    'X-Appwrite-Key': process.env.APPWRITE_API_KEY
+}
+
+const delay = ms => new Promise(res => setTimeout(res, ms));
+add_data = {
+    "action": "addAnnouncements",
+    "announcements": [
+        {"title": "Message 7", "content": "Message 7 added from HTTP request"},
+        {"title": "Message 8", "content": "Message 8 added from HTTP request"}
+    ]
+}
+const addAnnouncements = async () => {
+    try {
+        const res_pos = await axios.post(
```

Comment:
what is `res_pos`?
Tests are no less important pieces of software than the actual feature. Quality is important for tests like for anything else.

also `res_get` and all occurrences below like line 60, 90, 125